systemtests/src/main/java/io/apicurio/registry/systemtest/framework/Constants.java

```diff
@@ -0,0 +1,57 @@
+package io.apicurio.registry.systemtest.framework;
```

Comment:
All the constants are extremely long and extremely hard to use. I would rather see some Environment class that loads all the variables at the start of the test suite. And there could be ternary operators which could add the defaults.

Reply:
Something like this https://github.com/EnMasseProject/enmasse/blob/development/systemtests/src/main/java/io/enmasse/systemtest/Environment.java Where all envs are in the same place.